_posts/languages/nodejs/2000-01-01-start.md

```diff
@@ -26,6 +26,10 @@ If a `yarn.lock` file is present at the root of your project, yarn will be used
 to install the dependencies and run scripts, otherwise npm will build the
 dependencies during the deployment.
 
+If a `package-lock.json` file is present at the root of your project and you
+are using npm version 6 or higher, Scalingo will use `npm ci` to install your
```

Comment:
Using long version `clean-install`  in the doc?

Reply:
Everybody use ci in their docs (heroku and NPM themself)

Reply:
Maybe between parenthesis, for isntance I didn't know what was `ci`, I thought weird, Continuous Integration? wtf I had to run `npm ci --help` on my laptop. That's why I'm proposing to explicit what it is like:

```
[...] Scalingo will use `npm ci` (for `clean-install`) to install your [...]
```